python/ola/OlaClient.py

```diff
@@ -87,23 +87,33 @@ def __repr__(self):
                     enabled=self.enabled)
 
   def __lt__(self, other):
+    if other.__class__ is not self.__class__:
+      return NotImplemented
     return self.id < other.id
 
   def __eq__(self, other):
+    if other.__class__ is not self.__class__:
+      return False
     return self.id == other.id
 
-  # These 4 could be replaced by functools.total_ordering when support
```

Comment:
Any reason for dropping the comment?

Reply:
No, by the way, do we still support python 2.6 ? The PSF does not support it anymore.

If we don't, I will happily remove the `le`, `gt` and `ge` methods from everywhere and put `@functools.total_ordering` on these classes.

Reply:
Oops, no, I remind now, functools.totalordering does only support `NotImplemented` since python 3.4. So we can't use it here until support for 2.7 is dropped.

Reply:
https://docs.python.org/3.7/library/functools.html#functools.total_ordering

Reply:
The 0.10 branch does, so I'd like to keep it, as this is only nominally a bug-fix. Please feel free to drop them from master and switch to functools.

Reply:
Dropping 2.7 isn't going to happen for a long time, Ubuntu Disco still has it:
https://packages.ubuntu.com/search?keywords=python&searchon=names&exact=1&suite=disco&section=all

As does 18.04:
https://packages.ubuntu.com/search?suite=bionic&exact=1&searchon=names&keywords=python

Which means it's live there until at least 2023!

Shame, given 2.7 supports it, but without NotImplemented:
https://docs.python.org/2.7/library/functools.html#functools.total_ordering